content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -324,6 +322,18 @@ For a reference to old feature gates that are removed, please refer to
 | `WatchBookmark` | `false` | Alpha | 1.15 | 1.15 |
 | `WatchBookmark` | `true` | Beta | 1.16 | 1.16 |
 | `WatchBookmark` | `true` | GA | 1.17 | - |
+| `WindowsEndpointSliceProxying` | `false` | Alpha | 1.19 | 1.20 |
+| `WindowsEndpointSliceProxying` | `true` | Beta | 1.21 | 1.21 |
+| `WindowsEndpointSliceProxying` | `true` | GA | 1.22| - |
```

Comment:
The above feature gate has been deleted and the docs are now moved to `content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md`. Please do not add it back.

Reply:
oops sorry, let me fix this up